src/docs/makeDocs/makeDocs.go

```diff
@@ -24,7 +20,7 @@ func main() {
 	dir := "/tmp" // place to save man pages
 
 	if argc > 2 {
-		fmt.Printf("ERROR: Too many arguments to %s\n", argv[1])
+		sylog.Errorf("Too many arguments to %s\n", argv[1])
```

Comment:
If we're going to call Error and then exit, we might as well use `sylog.Fatalf()` so that we exit with a status code indicating failure (applies to all these `sylog.Errorf` statements)

Reply:
Gotcha!